code/game/objects/items/weapons/clown_items.dm

```diff
@@ -238,43 +237,81 @@
 
 	if(isitem(target))
 		var/obj/item/target_item = target
+		if(target_item.current_glue_state != GLUE_STATE_NONE) //Check to see if its glued first.
+			to_chat(user,"<span class='warning'>Its already got glue on it!</span>")
```

Comment:
Oof this grammar, "its" is possessive and got is very informal. `It already has glue on it!` would be better.